
bool paused

string current_state
string[] state_history

string debug_information
